forum/serializers.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2017 Semester.ly Technologies, LLC
+#
+# Semester.ly is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Semester.ly is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+from rest_framework import serializers
+
+from forum.models import Transcript, Comment
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    author_name = serializers.CharField(source='author.get_full_name')
+
+    class Meta:
+        model = Comment
+        fields = (
+            'author_name',
+            'content',
+            'timestamp',
+            #'transcript',
+        )
+
+
+class TranscriptSerializer(serializers.ModelSerializer):
+    comments = CommentSerializer(many=True)
+
+    class Meta:
+        model = Transcript
+        fields = (
```

Comment:
would you want to also serialize advisors so UI can display them?

Reply:
@rbiz4 
My b, probably do want to have a list of advisor names

Reply:
Probably want semester to be the semester name instead of the id too